dbcon/joblist/tupleaggregatestep.cpp

```diff
@@ -5872,6 +5858,25 @@ uint64_t TupleAggregateStep::doThreadedAggregate(ByteStream& bs, RowGroupDL* dlp
   return rowCount;
 }
 
+bool TupleAggregateStep::cleanUpAndOutputRow(ByteStream& bs, RowGroupDL* dlp)
```

Comment:
I would change OutputRow part b/c we are sending a whole RGData worth 8192 rows to a next consumer.